include/mbgl/ios/MGLMapView.h

```diff
@@ -336,7 +349,13 @@ IB_DESIGNABLE
 *   @param mapView The map view that requested the annotation symbol name.
 *   @param annotation The object representing the annotation that is about to be displayed. 
 *   @return The marker symbol to display for the specified annotation or `nil` if you want to display the default symbol. */
-- (nullable NSString *)mapView:(MGLMapView *)mapView symbolNameForAnnotation:(id <MGLAnnotation>)annotation;
+- (nullable NSString *)mapView:(MGLMapView *)mapView symbolNameForAnnotation:(id <MGLAnnotation>)annotation __attribute__((unavailable("Use -mapView:imageForAnnotation:.")));
```

Comment:
Hmm, hadn't thought about Maki. You'd have to know which were in the style, though, as it's not all of them by default. I don't think there's much benefit, until such time that we maybe roll out an `RMMarker`-like constructor that lets you pass _any_ Maki symbol name (which it would then probably fetch on demand and upload via the new runtime imagery API).